crates/typst-library/src/foundations/dict.rs

```diff
@@ -266,6 +268,55 @@ impl Dict {
             .map(|(k, v)| Value::Array(array![k.clone(), v.clone()]))
             .collect()
     }
+
+    /// Produces a new dictionary with only the pairs from the original one for
+    /// which the value passes test.
```

Comment:
"passes the test" or passes `test` or maybe just "for which the given function returns true" for consistency with the array docs